doc/whats_new/v1.0.rst

```diff
@@ -105,10 +105,20 @@ Changelog
 :mod:`sklearn.inspection`
 .........................
 
-- |Fix| Allow multiple scorers input to
+- |Enhancement| Allow multiple scorers input to
   :func:`~sklearn.inspection.permutation_importance`.
   :pr:`19411` by :user:`Simona Maggio <simonamaggio>`.
 
+- |Enhancement| in :func:`~sklearn.inspection.plot_partial_dependence`, allow
+  `kind` to accept a list of strings to specify for each feature interaction
+  which type of plot a user wants to draw.
+  :pr:`19438` by :user:`Guillaume Lemaitre <glemaitre>`.
+
+- |Fix| in :func:`~sklearn.inspection.plot_partial_dependence`, be lenient and
+  only raise a warning when passing `kind='both'` or `kind='individual'` with a
+  mix of 1- and 2-way partial dependence in `features`.
```

Comment:
```suggestion
- |Fix| in :func:`~sklearn.inspection.plot_partial_dependence`, instead of raising
  an error, asking for ICE plots (`kind='both'` or `kind='individual'`) on a 2-way
  plot will use regular PDP and raise a warning.
```

Reply:
+1 for accepting this suggestion.